sdk/python/kfp/containers/_component_builder.py

```diff
@@ -219,6 +262,9 @@ def build_python_component(
   if python_version not in ['python2', 'python3']:
     raise ValueError('python_version has to be either python2 or python3')
 
+  if python_version == 'python2':
+    warnings.warn('Python2 is not longer supported')
```

Comment:
nit: How about just raise an error? KFP SDK doesn't support python2 anymore since a long time ago.

Reply:
Sounds good. Actually let me remove the the `python_version` parameter and hard-code it to be `python3` from now on.